resources/sdk/purecloudjava/templates/Configuration.mustache

```diff
@@ -20,5 +36,245 @@ public class Configuration {
    */
   public static void setDefaultApiClient(ApiClient apiClient) {
     defaultApiClient = apiClient;
+    autoReloadConfig = defaultApiClient.getAutoReloadConfig();
+    logger = defaultApiClient.getLogger();
+    configFilePath = defaultApiClient.getConfigFilePath();
+    ApiClient.LoggingConfiguration loggingConfiguration = defaultApiClient.getLoggingConfiguration();
+    Configuration configuration = new Configuration();
+    configFileHash = configuration.getConfigHash();
+    configuration.applyConfigFromFile();
+    if (autoReloadConfig) {
+      Runnable runnable = () -> configuration.runConfigChecker();
+      Thread configCheckerThread = new Thread(runnable);
```

Comment:
This should be set to a daemon thread so it doesn't keep the application running when all other threads have finished